.github/workflows/tests.yml

```diff
@@ -143,3 +143,79 @@ jobs:
       run: |
         python ./packages/schema/generate-spec.py -f markdown > spec.md
         diff -u ./packages/schema/jupyterwidgetmodels.latest.md ./spec.md
+  ui-test:
+    name: Visual Regression
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}-${{hashFiles('**/requirements.txt')}}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install jupyterlab==3.0.3
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Get yarn cache
+      id: yarn-cache
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.yarn-cache.outputs.dir }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+
+    - name: Build and Install ipywidgets
+      run: |
+        yarn install --frozen-lockfile
+        yarn run build
+    - name: Build the extension
+      run: |
+        pip install .
+        jlpm install
+        jlpm run build
+        cd jupyterlab_widgets
+        pwd
+        pip install -e .
+        jupyter labextension develop . --overwrite
+        jupyter labextension list
+    - name: Install Galata
+      run: |
+        cd ui-tests
+        npm install
+    - name: Launch JupyterLab
+      run: |
+        cd ui-tests
+        npm run start-jlab:detached
```

Comment:
Since `yarn` is currently used for the dev setup of ipywidgets, should we also use it for the `ui-tests` directory?

Reply:
Just pushed 1fe8d6d1366239e278929a6e9d6857f3fcc7fb60 for this.